doc/onboarding-storage-provisioners.md

```diff
@@ -0,0 +1,22 @@
+# Onboarding a storage provider to CDI
+
+## Introduction
+
+As explained extensively in the [storage profile](./storageprofile.md) documentation, CDI keeps internal knowledge  
+about a wide range of storage providers in order to provide defaults & hints as they are needed.  
+This doc is about "educating" CDI about a new storage provider.
+
+## StorageProfile.Spec fields
+In order to onboard a storage provider, one should be able to pick the correct values for the storage profile parameters.  
+Detailed explanations and tips can be found in the storage profile's parameters [section](./storageprofile.md#parameters).
+
+For example, for ceph rbd, CDI knows that:
+- These are the access/volume modes (in this preference order) supported
+https://github.com/kubevirt/containerized-data-importer/blob/c6089cbcb01ab58e75b25be1770ce0f3f4625014/pkg/storagecapabilities/storagecapabilities.go#L255-L261
+- The preferred clone strategy is csi-clone (internally creates a temporary snapshot)
```

Comment:
"internally creates a temporary snapshot" might be confusing to someone who doesn't already know this.  I would leave it out.  Instead, you could mention something like "Of the three cloning strategies (csi-clone, snapshot, and copy), the preferred strategy is csi-clone.

Reply:
Fair, dropped it. We already state csi-clone is preferred in the storage profile doc